paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -31,9 +31,12 @@ import app.cash.paparazzi.accessibility.RenderSettings.DEFAULT_TEXT_SIZE
 import app.cash.paparazzi.accessibility.RenderSettings.getColor
 import app.cash.paparazzi.accessibility.RenderSettings.toColorInt
 import app.cash.paparazzi.accessibility.RenderSettings.withAlpha
+import java.util.concurrent.atomic.AtomicInteger
 
 class AccessibilityRenderExtension : RenderExtension {
 
+  private val nextGeneratedId = AtomicInteger(1)
```

Comment:
another option, instead of having the renderExtension manage the state, we can move this to RenderSettings and update in getColor, as suggested below.

If we do that though, then either:
1) RenderSettings should no be longer be an object, since it's stateful
2) or workaround this, by adding 

  fun resetGeneratedId() = nextGeneratedId.set(1)

and calling resetGeneratedId() towards the end of AccessibilityRenderExtension.renderView